provider/bigquery.go

```diff
@@ -870,16 +876,20 @@ func (store *bqOfflineStore) RegisterResourceFromSourceTable(id ResourceID, sche
 }
 
 func (store *bqOfflineStore) RegisterPrimaryFromSourceTable(id ResourceID, tableLocation pl.Location) (PrimaryTable, error) {
+	store.logger.Infow("Registering primary from source table", "id", id.Name)
```

Comment:
logger := store.logger.WithValues(map[string]interface{"id": id.Name, ....
logger.Infow("Registering primary)

Aso don't forget to log the location adn the variant, might as well log the whole ResourceID